APISpecification.yaml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.0
+info:
+  title: Tools api
+  version: 0.0.1
+paths:
+  /api/tools/vep/submissions/{id}:
+    get:
+      description: Returns results of VEP analysis
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: page
+          in: query
+          required: false
+          schema:
+            type: number
+            default: 1
+        - name: per_page
+          in: query
+          required: false
+          schema:
+            type: number
+            default: 100
+      responses:
+        '200':
```

Comment:
Does client expect 404 when results ID is not found?

Reply:
Yes; but going back to our discussion during code review today, isn't this standard behaviour? For example, I am sure client can also send an invalid payload (e.g. request a minus first page); but we don't expect the submitter of the spec to describe the 422 error?

Reply:
Error responses are just like 200:
- these need to be implemented in BE side (e.g. specific inspection and handling of payloads from downstream (404) or upstream (422))
- these need to be agreed with and handled/implemented in client side
No problem if we leave 400/422 blank: it means no need to implement these for now.